src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useState } from 'react';
+import produce from 'immer';
+import './index.css';
+
+
+function App() {
+
+  const [data, setData] = useState('');
+
+  const handleChange = (e) => {
+    const data = e.target.data
+    setData(data)  
+  };
+
+ 
+
+
+  return (
+    <div id="paper">
+      <div id="pattern">
+        <div id="content">
+          <textarea 
+            value={data} 
+            onChange={handleChange}
+            className="textbox" 
+            />        
+           
+          </div>
+      </div>
+
+    </div>
+  );
+}
+
+export default App;
```

Comment:
Really impressed with how much you got done during the time allotted. If you want to keep working on this, you could change data to an array, add the textarea to a form and add a submit button as well. Then handle onSubmit instead of onChange, and push the value of the textarea to data when it's called.

Just be sure to use setData like this in the submitHandler:
```js
// You probably already know this :)
setData([...data, newString])
```

So that it's still reactive